arroyo-worker/src/operators/sources/kafka/mod.rs

```diff
@@ -184,7 +184,12 @@ where
                             }
                         },
                         Err(err) => {
-                            error!("encountered error {}", err)
+                            ctx.control_tx.send(
```

Comment:
This should probably also capture some of the fatal errors, like failing to deserialize a message (line 180) or failing to create the consumer (167)

Reply:
👍 done